README.md

```diff
@@ -80,7 +174,7 @@ Additionally to all the methods declared in the [Parser-API](https://github.com/
 
 Direct access to the parsed JSON document is always available through the `doc.json()` method.
 
-See [API documentation](/docs/api.md) for more example and full API reference information.
+See API documentation for more example and full API reference information.
```

Comment:
One of the workflows is showing us errors regarding the link with in this paragraph and I thought that by removing it the workflow would go through but it didn't I would leave it as is, because this section needs to be corrected anyway.